apps/main/src/dao/components/PageGauges/GaugeListItem/index.tsx

```diff
@@ -35,7 +37,13 @@ const GaugeListItem = ({
   powerUsed,
   userGaugeVote = false,
   addUserVote = false,
+  userAddress,
```

Comment:
if you make this optional you don't have to pass `userAddress=''` in the parent component using this component
```suggestion
  userAddress = '',
```